lib/reverse_sentence.rb

```diff
@@ -1,6 +1,44 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n). N is the length of the input string.
+# Space complexity: O(n) for words array. N is number of characters in the input string.
+
 def reverse_sentence(my_sentence)
```

Comment:
Could you dry this up a bit?  Notice that you're reversing twice.  Could you create a method to reverse a string between two indices?